code/src/index.css

```diff
@@ -5,9 +5,109 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #000;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+.movie-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+}
+
+.list-link {
+  max-width: 25%;
+  height: auto;
```

Comment:
If you use width instead of max-width you will get the movies aligned nicely in the movielist 🍿

Reply:
Spot on! Thanks for pointing that out, now I got rid of that annoying styling error 👍